content/docs/typechecking-with-proptypes.md

```diff
@@ -143,9 +143,9 @@ MyComponent.propTypes = {
 };
 ```
 
-### Default Prop Values {#default-prop-values}
+### Valores Default de Props {#default-prop-values}
```

Comment:
what if we keep with both translated/original versions here? 

```diff
-Valores Default de Props
+Valores Padrão de Props (Default Prop Values)
```

Reply:
got it. I was trying to figure out a good translation for `default` and couldn't find one. I kept the `default` word but as you mentioned, translating and putting on parenthesis seem better. When using parenthesis on Crowdin I got a warning